submission/proxy.go

```diff
@@ -112,6 +116,9 @@ func (p *Proxy) Run(ctx context.Context, llRefresh time.Duration, rootsRefresh t
 			case llData := <-p.llWatcher.LLUpdates:
 				if err := p.restartDistributor(ctx, llData.List); err != nil {
 					p.Errors <- err
+				} else if !init {
+					init = true
+					p.Init <- true
```

Comment:
I wonder if the channel should be closed at this point?

Reply:
Closed